blt/src/Commands/HumsciConfigCommand.php

```diff
@@ -11,6 +11,8 @@
  */
 class HumsciConfigCommand extends ConfigCommand {
 
+  use HumsciTrait;
```

Comment:
Shouldn't traits be namespaced as well?

Reply:
I assumed it would need to be namespaced the same as the classes since the classes had to be namespaced something specific. this trait has namespace `Acquia\Blt\Custom\Commands`